src/lib.rs

```diff
@@ -64,6 +64,7 @@ pub fn generate(ctx: &RenderContext) -> Result<(), Error> {
         None => ctx.destination.join("book.epub"),
     };
 
+    std::fs::create_dir_all(&ctx.destination)?;
```

Comment:
Should this be wrapped in a `if !ctx.destination.exists()` so we don't unnecessarily try to create all parent directories if `ctx.destination` already exists?

Reply:
Done!